bigbluebutton-html5/imports/ui/components/chat/chat-graphql/chat-message-list/page/chat-message/message-content/poll-content/component.tsx

```
@@ -89,32 +89,49 @@
+            />
+            <YAxis width={100} type="category" dataKey="pollAnswerWithNumVotes" tick={<CustomizedAxisTick />} />
+            <Bar dataKey="numVotes" fill="#0C57A7" />
+          </BarChart>
+        </ResponsiveContainer>
+      </Styled.PollWrapper>
+      <table className="sr-only" id={`poll-result-${pollData.id}`}>
+        <caption>{pollData.questionText}</caption>
+        <tbody>
+          {translatedAnswers.map((ans: Answers & { pollAnswer: string }) => {
+            return (
+              <tr>
```

Biome (1.9.4)

[error] Missing key property for this element in iterable.

The order of the items may change, and having a key can help React identify which item was moved.
Check the React documentation. 

(lint/correctness/useJsxKeyInIterable)